js/index.js

```diff
@@ -0,0 +1,16 @@
+import { myDecipher } from './EncryptData.js'
+
+document.getElementById("login").onsubmit = function () {
+    let account_hash = document.getElementById("account_hash_input").value
+    let account_password_in_hash = JSON.parse(myDecipher(account_hash)).user_data.user_password
+    let password = document.querySelector("#password_input").value
+
+    if (password == account_password_in_hash) {
+        let data_for_ls = myDecipher(account_hash)
+        localStorage.setItem("data", data_for_ls)
```

Comment:
i think that it will be ok if we create new method in this file which will check new localstorage for correctness.
we need this coz we can get wrong hash from user
so u must create comething which will be check all fields which we need obviously